### DISCUSSION TITLE
Perl::LanguageServer 2.5

### DISCUSSION THREAD

**Discussion by JaysonWonder** [microsoft/vscode-discussions#443] [Extension Development QnA] [answered]
How do I get this enabled on windows with Strawberry Perl 5.32? The extension will install but it will not start or run. Is there still support for this language?

**Comment by gjsjohnmurray** [comment, marked as answer]
Please contact the extension author. This discussion forum is for extension **developers** rather than users.

**Reply by JaysonWonder** [reply]
Thank you and sorry

**Comment by TeddyRistainoi** [comment]
To enable the extension on Windows with Strawberry Perl 5.32, you may need to make sure that the extension is added to the system's path environment variable and that all dependencies are installed. Additionally, you can check the documentation or forums for the language to see if there is any specific support or troubleshooting advice available.
